prowler/providers/aws/services/rds/rds_instance_transport_encrypted/rds_instance_transport_encrypted.py

```diff
@@ -42,15 +42,15 @@ def execute(self):
                     for parameter in db_instance.parameters:
                         if (
                             parameter["ParameterName"] == "rds.force_ssl"
-                            and parameter["ParameterValue"] == "1"
+                            and parameter.get("ParameterValue", "0") == "1"
```

Comment:
Why this is not done at the service level?

Reply:
Because in the service we are storing all the parameters in an attribute. It is not like the DB Cluster.

Reply:
okay, it'd be better to do this on the service side but it's good for now.